user-guide/modules/ROOT/pages/boost-test-tutorial.adoc

```diff
@@ -13,11 +13,11 @@ Let's get started with a trivial example.
 
 Here's a very simple example of a test suite with Boost.Test:
 
-. Use Visual Studio to create a C++ Console application, call it something like "StringRev".
```

Comment:
The tutorial is quite nice. I don't think most people are on Visual Studio though (although Vinnie is). A lot of people use VSCode, jetbrains, etc... and even Visual Studio itself is moving towards CMake, so these workflows are kind of unmaintained and are going to be deprecated at some point. 

I think the best option would be to create some tabs at these steps where we are setting things up with "CMake", "Visual Studio", "By Hand", like in the "getting started" page. 

If that's too much work for now and we only want to have one option, this second best alternative for a single option would be CMake, because it's the de facto build system for C++ at the moment. You can make local tests with CMake directly from VS so you don't even have to change your workflow.

Reply:
@alandefreitas OK, perhaps a VSC and CMake option too, I'll investigate.

Reply:
When you create a project, VS lets you choose between a cmake project and a "legacy" VS solution, which is now considered just one possible cmake target. Just replicated the pattern from "Getting Started" to link the libraries in CMake and all the rest of the content should be exactly the same.